### ISSUE TITLE
[SUGGESTION] Add the ability to disable the dispatcher

### ISSUE THREAD

**Issue by Eldrinn-Elantey** [mamoniot/project-cybersyn#25]
It would be convenient if there was a button or keyboard shortcut that allows you to disable the dispatcher completely on all surfaces

**Comment by Dimava** [comment]
Was not there a per-map setting for that?

**Comment by Eldrinn-Elantey** [comment]
I know that you can change the parameter to 0 and it will be the same but it is still more convenient when there is a separate key combination and an alert that the manager is turned off

**Comment by Ashtoruin** [comment]
Yeah. I sometimes find myself missing the keyboard shortcut to disable it when I need to upgrade the equipment in my trains.
